server/src/test/java/com/adobe/testing/s3mock/its/S3TestBase.java

```diff
@@ -108,14 +108,18 @@ public void cleanupFilestore() {
     }
   }
 
-  private String getHost() {
+  String getHost() {
     return System.getProperty("it.s3mock.host", "localhost");
   }
 
   private int getPort() {
     return Integer.getInteger("it.s3mock.port_https", 9191);
   }
 
+  int getHttpPort() {
+    return Integer.getInteger("it.s3mock.port_http", 9090);
```

Comment:
This needs to be added in server/pom.xml, after line 218:
`<it.s3mock.port_http>${it.s3mock.port_http}</it.s3mock.port_http>`